go/exocom/features/register-service.feature

```diff
@@ -23,13 +23,12 @@ Feature: Manage new instances of services
     Then ExoCom should have the config:
       """
       {
-        "clients": {
-          "users": {
+        "clients": [
+          {
             "role": "users",
-            "serviceType": "users",
-            "internalNamespace": "foo"
+            "serviceType": "users"
           }
-        },
+        ],
```

Comment:
Updated these as the key is redundant and the internalNamespace is available in the routes